### DISCUSSION TITLE
My Documents are not Recognized due to many bugs and glitches

### DISCUSSION THREAD

**Discussion by Fenderus** [community/community#69200] [GitHub Education]
### Hi everyone,

I have been trying to apply for "Github Education" for a long time, and each time I try, there are always errors, bugs and glitches. And here they are:

1. Back ("Main") Camera of phone doesn't get recognized at all:
    - I tried using my Oppo A17K and my Father's IPhone X to apply, but every time I do it, I had to use the front "Selfie Camera", 
      because when clicking on the Back Camera Option, it doesn't work. Which 
      means I had to curve my neck a lot of times to get the picture to align, but with every attempt, There are recognition errors. And 
      after doing that for a while, I got serious neck pain.
2. Image gets blurry when taking a photo:
    - I tried to use everything I can to take a photo of my documents, like using OBS's Virtual camera and using an image on the 
      desktop, or using the phone. But with every attempt, the image get's blurred after processing the image which gave more 
      recognition errors.
3. Even when getting a good and solid photo without the errors, It goes back to the "Upload Proof" Page without any updates.
4. Github says that "Individual manual reviews and status updates are no longer provided" in Github support.

Yeah, it's literally blocking students from doing anything probably. Thanks for the answers in advance.

**Comment by samus-aran** [comment]
Hi @Fenderus 

Thank you for sharing your experience. We have passed this along to our internal team regarding to the front camera limitation. 

What we would suggest is to use an alternative piece of evidence e.g. Student ID that would allow you to take a detailed image so that the text is legible. If possible, to assist with session validation, try and take a partial photo of a dated piece of evidence alongside or layered with the Student ID. You want to include the most important elements of the evidence e.g. school name, your name, session date and try and focus on those elements so the text is legible. Try not to use software to resize or combine the photos as this will be penalised.

Let me know when you re-apply so I can assist 👍

**Reply by Fenderus** [reply]
I currently have a student transcript in SF10 form and my previous grade ID which might not be valid, would the two work in one picture?

**Reply by FrozenPyrozen** [reply]
@ErraqiNouhaila the same here, photo from front camera in bad quality condition, but when I upload photo from phone got errors:

Please correct your GitHub billing information with your last name exactly as it appears in your academic affiliation document.
Please correct your GitHub billing information with your first name exactly as it appears in your academic affiliation document. Do not use a variation of your name or a nickname.
